helpers/store/composer.js

```diff
@@ -1,43 +1,89 @@
+import commonHelper from '~/helpers/common'
+import knitLogger from '~/config/logger'
 import _ from 'lodash'
 
-const methods = {
-  cancel () {
-    throw new Error('cancel')
+function _validateComposableInputs (name, caller) {
```

Comment:
Why do you use underscore now as name? I suppose you try to express it as internal, should not be exported? Is it really needed? Ping @zrebcu411

Reply:
Yes I'm trying to articulate that it's internal and should not be exported. I think yes it's needed and we should validate the inputs in order to create logs, and act properly.